update-tests.sh

```diff
@@ -5,6 +5,7 @@ set +o errexit
 
 for test in $(find tests -name '*.ts'); do
   node src/index.js $test > $test.out 2>&1
+  rc=${PIPESTATUS[0]}; if [[ $rc != 0 ]]; then exit $rc; fi
```

Comment:
Presently it'd continue to run the git commands even if the test could not be completed. This addresses that, just existing with error code if it failed.